### PR TITLE
BRS 8 - Full SSR Hard Source and Loadable support

### PR DESCRIPTION
**Pull request by olliecurtis**
For more context see #77 .

This PR is to align #77 to BRS v8 for testing before further exploration of configurability for our consumers.

### CODE REVIEW

**Comment by olliecurtis** [comment]
The travis build is expected to fail here as the app paths have been changed in the config.js to support SSR at a particular run which for non SSR consumers modifying these paths would mean that they don't get generated properly so would need to look at the impact on that when people use BRS and serve their website

**Comment by ajones513** [comment]
Diff locals perfect @olliecurtis, nice spot with the CSS `/locals` package change. Next step as you say is to make the breaking changes opt-in.

File: packages/react-scripts/scripts/start-ssr.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+const configFactory = require('../config/webpack.config.ssr');
+const paths = require('../config/paths');
+
+const statusFile = require('./utils/statusFile');
+
+const config = configFactory('development');
+const appName = require(paths.appPackageJson).name;
+const useYarn = fs.existsSync(paths.yarnLockFile);
+
+const compiler = createCompiler(webpack, config, appName, undefined, useYarn);
```
Comment (olliecurtis) [inline packages/react-scripts/scripts/start-ssr.js:47]: Turns out the params of this function have changed to accept an object of these parameters so will update with the correct expected format for this

See here for example: https://github.com/Skyscanner/backpack-react-scripts/blob/9ab25eb68904847805d4a5681ab7e6a8620f0845/packages/react-scripts/scripts/start.js#L115

**Comment by olliecurtis** [comment]
As BRS 9 is the current supported version and BRS 10 is on the way closing this PR for now

**Pull request closed without merging**